roles/maps/tasks/install.yml

```
@@ -4,6 +4,13 @@
     state: directory
     # mode: 0755
 
+- name: Install aria2
+  apt:
+    name:
+      - aria2c
+
+    state: present
```

IMHO these 4 lines can be tightened up to just 1:

```suggestion
    name: aria2c
```